openapi/swagger.json

```diff
@@ -896,6 +905,91 @@
       }
     },
     "/tables/{tableID}/rows": {
+      "head": {
```

Comment:
Can we document a successful table version response, including the ETag response header? Right now the docs only show a generic error response (not even sure what error case is possible with this call).